CODEOWNERS

```diff
@@ -8,3 +8,4 @@
 /src/v/coproc/             @graphcareful @dotnwat
 /src/v/pandaproxy/         @benpope @dotnwat
 /src/v/raft/               @senior7515 @dotnwat @mmaslankaprv
+/src/v/js/                 @0x5d
```

Comment:
i think you should also add yourself so that you are pinged for code reviews that touch that direcotry

Reply:
Oh sounds good